pom.xml

```diff
@@ -76,6 +76,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jsch</artifactId>
+            <!-- TODO https://github.com/jenkinsci/jsch-plugin/pull/33 -->
+            <version>0.2.8.63.vf67c9d214a_59</version>
```

Comment:
Requires https://github.com/jenkinsci/jsch-plugin/pull/33

Otherwise need to re-implement with trilead ssh